client/pages/home.jsx

```diff
@@ -1,10 +1,10 @@
 import React from 'react';
-import HelloWorld from '../components/hello-world';
+import BusinessProfile from '../components/business-profile';
 
 export default function Home(props) {
   return (
     <div>
-      <HelloWorld />
+      <BusinessProfile />
     </div>
   );
 }
```

Comment:
Instead of putting your business profile in your home page, you can render it directly in App.